textgrad/engine/__init__.py

```diff
@@ -14,7 +15,10 @@ def get_engine(engine_name: str, **kwargs) -> EngineLM:
     if "seed" in kwargs and engine_name not in ["gpt-4", "gpt-3.5"]:
         raise ValueError(f"Seed is currently supported only for OpenAI engines, not {engine_name}")
 
-    if (("gpt-4" in engine_name) or ("gpt-3.5" in engine_name)):
+    if "qwen2" in engine_name:
```

Comment:
I’d check if “ollama” is in engine name (again in case we use qwen from some other provider)